apps/prairielearn/src/question-servers/freeform.ts

```diff
@@ -1701,50 +1731,40 @@ async function getCachedDataOrCompute(
   // cacheKey and either return the cachedData for a cache hit,
   // or compute the cachedData for a cache miss
   const getFromCacheOrCompute = async (cacheKey: string) => {
-    let cachedData;
+    let cachedData: unknown;
```

Comment:
```suggestion
    let cachedData: unknown | null;
```

This makes it a little more clear to me the signature of `cache.get`.

Reply:
TIL that unknown absorbs any other type: https://stackoverflow.com/a/79528531/5684541